testThreeJS.html

```diff
@@ -1,359 +1,378 @@
-			{
-				var loader = new THREE.VTKLoader();
-				loader.load( aFile, function ( geometry ) {
-
-					geometry.computeVertexNormals();
-					geometry.translate(-center.x, -center.y, -center.z);
-
-					var material = new THREE.MeshLambertMaterial( { color: aColor, side: THREE.DoubleSide, transparent: transparentON, wireframe: wireframeON} );
-					material.opacity = anOpacity;
-					var mesh = new THREE.Mesh( geometry, material );
-					mesh.name = aName;
-
-					scene.add( mesh );
-				} );
-			}
-			
-			function animate() {
-
-				requestAnimationFrame( animate );
-
-				controls.update();
-				renderer.render( scene, camera );
-
-				stats.update();
-			}
-
-		</script>
-		
-		<script>
-			function toggleVisibility(objectname) {
-
-				// retrieve scene
-				scene = eval('scene');
-				object = scene.getObjectByName(objectname);
-				
-				// toggle visibility
-				object.visible = !object.visible;
-				
-				// update checkbox
-				document.getElementById(objectname + '_checkbox').checked = object.visible;
-			}
-		</script>
-
-		<script>
-			function moveCamera(angle) {
-				
-				var dx = Math.pow(camera.position.x - cameraFocal.x, 2);
-				var dy = Math.pow(camera.position.y - cameraFocal.y, 2);
-				var dz = Math.pow(camera.position.z - cameraFocal.z, 2);
-
-				var distCamObj = Math.sqrt(dx + dy + dz);
-				
-				camera.position.x = cameraFocal.x + distCamObj*Math.cos(angle*Math.PI/180);
-				camera.position.y = cameraFocal.y + distCamObj*Math.sin(angle*Math.PI/180);
-				camera.position.z = cameraFocal.z;
-				
-				camera.up.x = 0;
-				camera.up.y = 0;
-				camera.up.z = 1;
-			}
-		</script>
-	</body>
+        <head>
+                <title>three.js webgl - loaders - vtk loader</title>
+                <meta charset="utf-8">
+                <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+                <style>
+                        body {
+                                font-family: sans-serif;
+                                background-color: #000;
+                                color: #fff;
+                                margin: 0px;
+                                overflow: hidden;
+                                display: inline;
+                        }
+                        /*#info {
+                                color: #fff;
+                                position: absolute;
+                                top: 10px;
+                                width: 100%;
+                               text-align: center;
+                                z-index: 100;
+                                display:block;
+                        }
+                        #info a, */.button { color: #f00; font-weight: bold; text-decoration: underline; cursor: pointer }
+
+                        .block {
+                        text-align: left;
+                        line-height: 40px;
+                        position:absolute;
```

Comment:
"block" (vertical buttons) and "scene" (the heart) are placed with "absolute" parameter